lib/svg-preview-view.js

```diff
@@ -81,9 +81,9 @@ class SvgPreviewView extends View {
     this.controls.find('a').on('click', (e) => {
       this.changeBackground($(e.target).attr('value'))
     })
-    this.zoomOutButton.on('click', (e) => this.zoom(-0.1))
+    this.zoomOutButton.on('click', (e) => this.zoom(9 / 10 - 1))
```

Comment:
The value of the formular is the same. I only updated it to match the zoom-in formular.